www/js/init.js

```diff
@@ -55,6 +55,8 @@
  * @property {string} cacheIDB - Name of the Indexed DB database
  * @property {boolean} isFileSystemApiSupported - A boolean indicating whether the FileSystem API is supported.
  * @property {boolean} isWebkitDirApiSupported - A boolean indicating whether the Webkit Directory API is supported.
+ * @property {boolean} useLibzim - A boolean indicating weather to use the libzim to load zim files.
+ * @property {"wasm-dev" | "wasm" | "asm" | "asm-dev" | "default"} libzimMode - A value indicating which libzim mode is selected.
```

Comment:
I don't think these are types. You should probably just use `{String}` and probably use capital initial letter for types (I may be wrong, but I think that's what we normally use). You can list the acceptable strings in the description instead.

Reply:
These are the literal types a string could be anything but this property will only expect certain strings
![image](https://github.com/kiwix/kiwix-js/assets/47270636/93866fa0-727b-45b1-be61-326206129917)

Reply:
You are listing permitted properties, but the actual type is `{String}`. A value's type and its permitted values are not the same thing in my books. But I may be wrong, and this could be as JSDoc convention. Can you point me to any JSDoc documentation that this is the correct way to list accepted values? I think a parallel might be listing a type a `{Boolean}` as opposed to listing it as `{True | False | Null | 1 | 0 | "" | Undefined}`, because those are all permitted Boolean values.

Reply:
https://www.geeksforgeeks.org/what-are-string-literal-types-in-tpescript/
I could not find any proper article explaining string literal in JS but here is an article explaining using typescript.


If we use the type Boolean, we ought to stick to either true or false. However, because JS isn't type-safe, it doesn't enforce this, so we're treating truthy/falsy values as booleans here. The reason for preferring string literals over strings is their resemblance to enums. Rather than permitting any string value, we restrict it to specific strings as the type. When a new user edits this value, the editor will indicate the exact type, aiding the developer in understanding the precise values a particular variable might hold.

Reply:
OK, you've clearly researched this, thanks for the info. I learnt something.😉

Reply:
I would just add that as our convention throughout is to use single quotes, please change the use of double quotes to single here, otherwise it looks incongruous.